telemetry/convert.py

```diff
@@ -36,6 +37,8 @@ class Converter:
     """A class for converting incoming payloads to a more compact form"""
     VERSION_UNCONVERTED = 1
     VERSION_CONVERTED = 2
+    # the unified FHR/Telemetry ping has a top-level version of 2 and payload version of 4
+    VERSION_UNIFIED = 2
```

Comment:
I settled on the following approach:
- presence of "version" means this is a unified ping
- version 4 is the raw incoming unified ping (unconverted)
- version 5 is a converted unified ping. for saved-session pings, it's basically a hybrid ping that could pass for a classic ping. for other unified ping types, it's just a unified ping with an "info" section bolted on